source/contracts/libraries/Extractable.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity 0.4.17;
+
+import 'libraries/token/ERC20Basic.sol';
+import 'Controlled.sol';
+
+
+contract Extractable is Controlled {
+    // Send accidentally received ETH to the destination
+    function extractETH(address _destination) public onlyControllerCaller returns(bool) {
+        require(mayExtract(ERC20Basic(0)));
+        require(_destination.call.value(this.balance)());
+        return true;
+    }
+
+    // Send accidentally received tokens to the destination
+    function extractTokens(address _destination, ERC20Basic _token) public onlyControllerCaller returns (bool) {
+        require(mayExtract(_token));
+        uint256 _balance = _token.balanceOf(this);
+        _token.transfer(_destination, _balance);
```

Comment:
This should be wrapped in a `require()`.  ERC20 doesn't require throw on failure, it allows return false on failure and having the function clearly fail on-chain rather than silently fail would be good.